packages/js/src/utils/compiler-helper-dependency.ts

```diff
@@ -89,12 +117,23 @@ export function getHelperDependency(
   };
 }
 
-export function getHelperDependenciesFromProjectGraph(
+export async function getHelperDependenciesFromProjectGraph(
```

Comment:
suggestion: since this function is only used in `generate-package-json-webpack-plugin`, and the plugin seems to have access to the `ProjectGraph` already, I'd make this one accept a `projectGraph` instead of breaking it up and use `createProjectGraphAsync`.